app/data/events/sanitise-events.js

```diff
@@ -1,24 +1,25 @@
-const moment = require('moment')
-const { convertToPence } = require('../../currency-convert')
-const { PAYMENT_EXTRACTED } = require('../../constants/events')
-const { DATE } = require('../../constants/date-format')
+const moment = require('moment-timezone')
 const schemeNames = require('../../constants/scheme-names')
 const eventDetails = require('../../constants/event-details')
+const { PAYMENT_EXTRACTED } = require('../../constants/events')
+const { DATE } = require('../../constants/date-format')
+const { TIMEZONE } = require('../../constants/timezone')
```

Comment:
Maybe swap the order of these to be TIMEZONE, DATE and then PAYMENT_EXTRACTED to match below?